Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: help
+
+#
+# Configuration variables
+#
+VENVDIR?=.venv
+VENV = $(VENVDIR)/bin
+
+PYTHON3OK :=$(shell python3 --version 2>&1)
```

Comment:
PYTHON30K`? 
This seems unnecessary as Python3 (3.5.3) is a project prerequisite also shared with KFP. The number of people using Python 2 for ML projects should be negligible and Python 2 is no longer supported